packages/@aws-cdk/aws-eks/package.json

```diff
@@ -80,6 +80,7 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@aws-cdk/lambda-layer-kubectl-v22": "0.0.4",
```

Comment:
Should this depend on `v2.0.0`? I'm assuming that is the stable version?

Reply:
Yep you are right, was waiting for that to get released.